src/OpenTelemetry.Exporter.Zipkin/ZipkinExporterOptions.cs

```diff
@@ -22,25 +23,33 @@ namespace OpenTelemetry.Exporter.Zipkin
     /// </summary>
     public sealed class ZipkinExporterOptions
     {
-        /// <summary>
-        /// Gets or sets Zipkin endpoint address. See https://zipkin.io/zipkin-api/#/default/post_spans.
-        /// Typically https://zipkin-server-name:9411/api/v2/spans.
-        /// </summary>
-        public Uri Endpoint { get; set; } = new Uri("http://localhost:9411/api/v2/spans");
+        internal const string DefaultServiceName = "OpenTelemetry Exporter";
+
+#if !NET452
+        internal const int DefaultMaxPacketSize = 4096;
+#endif
 
         /// <summary>
-        /// Gets or sets timeout in seconds.
+        /// Gets or sets the name of the service reporting telemetry.
         /// </summary>
-        public TimeSpan TimeoutSeconds { get; set; } = TimeSpan.FromSeconds(10);
+        public string ServiceName { get; set; } = DefaultServiceName;
 
         /// <summary>
-        /// Gets or sets the name of the service reporting telemetry.
+        /// Gets or sets Zipkin endpoint address. See https://zipkin.io/zipkin-api/#/default/post_spans.
+        /// Typically https://zipkin-server-name:9411/api/v2/spans.
         /// </summary>
-        public string ServiceName { get; set; } = "OpenTelemetry Exporter";
+        public Uri Endpoint { get; set; } = new Uri("http://localhost:9411/api/v2/spans");
 
         /// <summary>
         /// Gets or sets a value indicating whether short trace id should be used.
         /// </summary>
         public bool UseShortTraceIds { get; set; }
+
+#if !NET452
+        /// <summary>
+        /// Gets or sets the maximum packet size in bytes. Default value: 4096.
+        /// </summary>
+        public int? MaxPacketSize { get; set; } = DefaultMaxPacketSize;
```

Comment:
Added `MaxPacketSize`. Zipkin really isn't sending packets, per se, it is sending http chunks, but I thought it might be nice to be consistent with JaegerExporter?

Reply:
Minor: consider `MaxPayloadSize`.

Reply:
Updated